qutip/configrc.py

```diff
@@ -38,6 +38,35 @@
     import ConfigParser as configparser #py27
 except:
     import configparser #py3x
```

Comment:
While this is nothing to do with your PR, we can remove the `try/except` and just have `import configparser` now, since we don't support Python 2.